src/LevelDBStore/IO/Data/LevelDB/Helper.cs

```diff
@@ -39,5 +41,14 @@ internal static byte[] ToByteArray(this IntPtr data, UIntPtr length)
             Marshal.Copy(data, buffer, 0, (int)length);
             return buffer;
         }
+
+        public static byte[] CreateKey(byte table, byte[] key = null)
```

Comment:
We should try to detach RocksDb form LevelDb

Reply:
? This method is only for LevelDb, and there is no dependence between them.